src/current/v23.2/changefeed-messages.md

```diff
@@ -17,6 +17,7 @@ This page describes the format and behavior of changefeed messages. You will fin
 - [Resolved messages](#resolved-messages): The resolved timestamp option and how to configure it.
 - [Duplicate messages](#duplicate-messages): The causes of duplicate messages from a changefeed.
 - [Schema changes](#schema-changes): The effect of schema changes on a changefeed.
+- [Filtering changefeed messages](#filtering-changefeed-messages): The settings and syntax to prevent and filter the messages that changefeeds emit.
```

Comment:
For both v23.2 and v24.1, I decided to add a small section to the changefeed message page that focuses on how you can filter/define/prevent messages from emitting. I included a short reference to cdc queries as well that links to the main reference page for that.

Reply:
Nice!